UI/window-basic-main.cpp

```diff
@@ -2471,15 +2449,26 @@ void OBSBasic::ReceivedIntroJson(const QString &text)
 		return;
 	}
 
+#if OBS_RELEASE_CANDIDATE > 0
+	constexpr const char *kLastInfoVersion = "InfoLastRCVersion";
+#elif OBS_BETA > 0
+	constexpr const char *kLastInfoVersion = "InfoLastBetaVersion";
+#else
+	constexpr const char *kLastInfoVersion = "InfoLastVersion";
+#endif
```

Comment:
The variables were initially declared outside of the function, I forgot to remove it when moving them inside (this is fixed now).

It's a common prefix for constant values in C++ codebases (I assume because in hungarian notation `c` is already taken for `char`). Apple also uses it for instance (e.g. in CoreFoundation https://developer.apple.com/documentation/corefoundation/core_foundation_constants?language=objc), and so does Google in their style guide: https://google.github.io/styleguide/cppguide.html#Constant_Names

As OBS does not have a consistent C++ style, or style guide for that matter, separate authors use their own preferred syntax. Me and a few others often use this prefix for global constants, as it makes it easy to distinguish a global const from a member or local variable, as well as from macros which are all-uppercase.